lecture10 template file/lecture10-alb.inport.yaml

```
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: lecture10 alb
+
+Resources:
+
+ alb:
+   Type: AWS::ElasticLoadBalancingV2::LoadBalancer
+   Properties: 
+     IpAddressType: ipv4
+     Name: lecture10
+     Scheme: internet-facing
+     SecurityGroups: 
+       - {'Fn::ImportValue': 'PublicSecurityGroup'}
+     SubnetMappings: 
+        - SubnetId: {'Fn::ImportValue': 'WebPublicSubnet1'}
+        - SubnetId: {'Fn::ImportValue': 'WebPublicSubnet2'}
+     Type: application
+     Tags: 
+       - Key: Name
+         Value: lecutre10
+ 
+ albTargetGroup:
+   Type: AWS::ElasticLoadBalancingV2::TargetGroup
+   Properties: 
+     VpcId: {'Fn::ImportValue': 'VPC'}
+     HealthCheckIntervalSeconds: 30
+     HealthCheckEnabled: true
+     HealthCheckPath: /
+     HealthCheckPort: traffic-port
+     HealthCheckProtocol: HTTP
+     HealthCheckTimeoutSeconds: 5
+     HealthyThresholdCount: 5
+     IpAddressType: ipv4
+     Port: 80
+     Protocol: HTTP
+     ProtocolVersion: HTTP1
+     TargetType: instance
+     Targets: 
+         - Id: {'Fn::ImportValue': 'WebInstance'}
+           Port: 80
+     UnhealthyThresholdCount: 2
+     Matcher: 
+        HttpCode: 200
+     Tags: 
+       - Key: Name
+         Value: lecture10 
+
+ Listener:
+  Type: AWS::ElasticLoadBalancingV2::Listener
+  Properties:
+     DefaultActions: 
+       - Type: forward
+         TargetGroupArn: !Ref albTargetGroup
+     LoadBalancerArn: !Ref alb
+     Port: 80
+     Protocol: HTTP
```

Line breakの警告が出ているので出ないようにする
```suggestion
     Protocol: HTTP

```

---

まだ変わらずのようです！

---

全yamlファイルの最終行に赤い警告がありましたが無くなりました。
指摘していただいた部分でしょうか？